docs/docs/general/options.md

```diff
@@ -2,9 +2,74 @@
 title: Options
 ---
 
+## Option resolution
+
+Options are resolved from top to bottom, using a context dependant route.
```

Comment:
I think either one is fine, but it looks like "dependant" is the british spelling and "dependent" is the american spelling, so the latter is more common

Reply:
Ok, our schools favored british english, at least in the 2nd millennium when I was learning :)